packages/webpack-config-single-spa/lib/webpack-config-single-spa.js

```diff
@@ -44,16 +44,18 @@ function webpackConfigSingleSpa(opts) {
           test: /\.(js|ts)x?$/,
           exclude: /node_modules/,
           use: {
-            loader: "babel-loader",
+            loader: require.resolve("babel-loader", { paths: [__dirname] }),
```

Comment:
We should remove these as devDependencies from the generator package.json templates now, since it's baked into webpack-config-single-spa

Reply:
I should have thought about that before. I agree.